DQMOffline/Lumi/src/TTrigger.cc

```diff
@@ -22,10 +22,12 @@ using namespace ZCountingTrigger;
 // the trigger can evolve and obtain a different trigger object name, but we still want this to 
 // be associated with the same leg (e.g. the trailing electron in a dielectron trigger)
 //
-TTrigger::TTrigger() { 
+TTrigger::TTrigger(std::vector<std::string> muonTriggerNames, std::vector<std::string> muonTriggerObjectNames) { 
```

Comment:
@davidwalter2 @andrius-k do we really want to pass these two vectors of strings by value? Why not by reference? 
May be as const references if they need to stay unchanged (as I imagine)?

Reply:
Ok thanks, I did another commit